src/main/java/io/codelex/enums/practice/Exercise2/Game.java

```diff
@@ -0,0 +1,79 @@
+package io.codelex.enums.practice.Exercise2;
+
+import java.util.Random;
+import java.util.Scanner;
+
+public class Game {
+    private static int playerWonCounter = 0;
+    private static int computerWonCounter = 0;
+
+    private static SPS computersChoice() {
+        Random random = new Random();
+        int randomNum = random.nextInt(3);
+        if (randomNum == 0) {
```

Comment:
var izmantot arī `SPS.values()` un so metodi arī varētu ielikt pasa enum klasē
`getRandomSPS()` for example